src/main.rs

```diff
@@ -1,3 +1,13 @@
+extern crate polychat_core;
+
+use std::env;
+
+use polychat_core::plugin::Plugin;
+
 fn main() {
-    println!("Hello, world!");
+    let plugin_path = env::args().nth(1).expect("Usage: ./polychat-basic <absolute path to library>");
+    let loaded_plugin = &mut Plugin::new("example", &plugin_path).expect("Could not load plugin");
+
+    let acc = loaded_plugin.create_account().expect("Could not create account for plugin");
+    loaded_plugin.print(acc).expect("Could not print account");
```

Comment:
So what is happening here? Is the UI going to be responsible for loading plugins?

Reply:
This is just a quick and dirty test of the build system, currently it is responsible for loading plugins, but the next iteration of `polychat-core` will start looking in a directory for plugins, and remove that from the UI's responsibilities.